### ISSUE TITLE
Everfrost Chip

### ISSUE THREAD

**Issue by Jindra555** [dalaranwow/dalaran-wow#4846]
https://github.com/dalaranwow/dalaran-wow/issues/4031

This bug seems to be still on live.

**Comment by Rushor** [comment]
Your issue posted here is a duplicate of issue: https://github.com/dalaranwow/dalaran-wow/issues/4031 (< click on this and you will be redirected to the original posted issue). 

As you can see the issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
